### ISSUE TITLE
Formatting fails with Prettier 1.19.1

### ISSUE THREAD

**Issue by ewendel** [jonlabelle/SublimeJsPrettier#194]
When [reporting an issue](https://github.com/jonlabelle/SublimeJsPrettier/issues), please include the following information in your post:

- [x] **Explain the Issue and Expected Behavior**
- [x] **Prettier version**
- [x] **JsPrettier Plug-in Version**
- [x] **Platform Details**
- [x] **Generated Prettier command line arguments**
- [x] **Is the same behavior observed when run against Prettier directly?**
- [x] **The contents of your** `User/JsPrettier.sublime-settings` **file**
- [x] **The contents of your** `<project_name>.sublime-project` **file (if applicable)**
- [x] **Steps to reproduce the behavior**

---

## Explain the Issue and Expected Behavior

The plugin fails to run Prettier, with the console showing `[error] regeneratorRuntime is not defined`.

## Prettier version

1.19.1

## JsPrettier Plug-in Version

1.28.0

## Platform Details

    - Sublime Text Version: 3.2.2
    - Sublime Text Build: 3211
    - Operating System Name: macOS
    - Operating System Version: Mojave

## Generated Prettier command line arguments

```
/Users/myuser/dev/myproject/node_modules/.bin/prettier 
    --stdin 
    --config /Users/myuser/dev/myproject/.prettierrc.json 
    --config-precedence cli-override 
    --parser typescript 
    --use-tabs false 
    --stdin-filepath /Users/myuser/dev/myproject/app/tabs/profile/screens/profile/Profile.tsx 
    --loglevel debug 
    --cursor-offset 1494
```

Prettier reported the following output:

```
[debug] normalized argv: {"_":[],"color":true,"editorconfig":true,"stdin":true,"use-tabs":false,"config":"/Users/myuser/dev/myproject/.prettierrc.json","config-precedence":"cli-override","parser":"typescript","stdin-filepath":"/Users/myuser/dev/myproject/app/tabs/profile/screens/profile/Profile.tsx","loglevel":"debug","cursor-offset":1494,"plugin-search-dir":[],"plugin":[],"ignore-path":".prettierignore","debug-repeat":0}
[error] regeneratorRuntime is not defined
```
## Is the same behavior observed when run against Prettier directly?

Nope. Running against Prettier directly outputs the file.

    prettier --stdin --stdin-filepath Profile.tsx --parser typescript --loglevel debug < Profile.tsx

## The contents of your `User/JsPrettier.sublime-settings` file

```
{
    "auto_format_on_save": true,
}
```

## The contents of your `<project_name>.sublime-project` file (if applicable)

N/A

## Steps to reproduce the behavior

1. Introduce faulty formatting in .js file
2. Save file
3. Expect file to reformatted, but isnt
4. The following is displayed in Sublime's bottom bar:  _"JsPrettier: Format failed! Open the console window to inspect errors."_

**Comment by jonlabelle** [comment]
Hi @ewendel. You'll need to jump over to the Prettier [issue board](https://github.com/prettier/prettier/issues) for potential causes and support, as this is not an error (`[error] regeneratorRuntime is not defined`) generated by, or because of the plug-in.

**Comment by ewendel** [comment]
Hi, @jonlabelle !

Thanks for creating and maintaining this plugin 🙏 

Could it be that this plugin is not calling Prettier correctly as of 1.19? 
It doesn't seem to me that the problem is on Prettier side, as it is working fine calling it directly from the CLI.

**Comment by ewendel** [comment]
Nevermind. It was a problem with multiple installatiosn of Prettier, as this is a problem with 1.19.0, fixed in 1.19.1 🙈
